### PR TITLE
Flow Improvement

### PR DESCRIPTION
**Pull request by Akshu-on-github**
I made the following changes to (hopefully) improve the flow of the document:
- Under `Curriculum`: Converted the three sentences into bullet points and rephrased the closing line
- Under `Learning Platform`: Rephrased the ways to improve the codebase
- Minor stuff: Added an exclamation point on line 6 and made a few modifications to lines 31-33

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX (Not to my knowledge)

Just a few minor changes - something to get me started in contributing

### CODE REVIEW

**Review by amaraneni265 — COMMENTED** [review]
Bullet points look good and structured compared to previous.

File: docs/index.md
Comment (nhcarrigan) [inline docs/index.md:23]:
```suggestion
**If you're interested in improving our curriculum, here's how: [How do I contribute to the curriculum?](how-to-work-on-coding-challenges.md)**
```
Hyperlink text ends with a `?` so no need to append a `.` 🙂 

I'm not 100% sure on the verbiage here, though.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
One slight nitpick here - the rest of the changes seem like an improvement to me 🥳

**Comment by AkshayaKulasekaran** [comment]
Good catch @nhcarrigan, I've commited the change!

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 I think I'm happy with this.

Congratulations on your first pull request to this project! We look forward to future contributions!

As an additional note, our workflow requires at least one additional approval before this can be merged.

**Comment by ojeytonwilliams** [comment]
@QuincyLarson, since you did the previous overhaul, do you want to take a quick look?

**Comment by QuincyLarson** [comment]
@AkshayaKulasekaran Thank you for opening your first pull requests. I looked at your edits, and I can understand the reasoning behind them. This said, I am happy with the original wording on this document, and I think it better reflects our organization's voice (not using exclamation marks, etc.)

**Pull request closed without merging**
